src/UCTNode.cpp

```diff
@@ -150,41 +137,34 @@ void UCTNode::link_nodelist(std::atomic<int> & nodecount,
     }
 
     // sort (this will reverse scores, but linking is backwards too)
```

Comment:
Comment could be updated. No more linking as in linked list.

Reply:
updated